applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_small_strain_umat_law.cpp

```diff
@@ -0,0 +1,31 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#include "custom_constitutive/small_strain_umat_2D_plane_strain_law.hpp"
+#include "custom_constitutive/three_dimensional.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(SmallStrainUMATLaw_HasReturnsTrueForCauchyStress, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Comment:
For completeness you could add:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(SmallStrainUMATLaw_HasReturnsTrueForCauchyStressAndStateVariables, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Reply:
After refactoring to a parameterized test, we have choosen another name together.